### DISCUSSION TITLE
Adding a photo gallery

### DISCUSSION THREAD

**Discussion by saeedesmaili** [adityatelange/hugo-PaperMod#1444] [Q&A]
I'm curious if anyone has been able to add a photo gallery to their Hugo+PaperMod site? I'm looking for something like [this](https://hugo-gallery-example.netlify.app/) to be used alongside my PaperMod theme, so the visitors of my website would see the gallery when visiting the `/photos` page. A few nice-to-have features of the gallery would be:

- Click to zoom on images
- Categorization / Image folders
- Integration with Flickr or AWS S3 to get images automatically

I'm interested in any idea on how to add such gallery to a Hugo+PaperMod site.

**Comment by russmckendrick** [comment]
While it doesn't fulfill 100% of your requirements I use the following on my site https://github.com/mfg92/hugo-shortcode-gallery see https://www.russ.foo/2023/12/23/my-top-10-of-2023/ for an example of how it looks within a post.

**Reply by saeedesmaili** [reply]
Thank you for sharing this! It's much nicer than [what I already have](https://saeedesmaili.com/photos/) which is a simple HTML embed of a Flickr album.

**Reply by jasonbraganza** [reply]
Came here to echo @saeedesmaili. Thank you so much for the pointer, @russmckendrick! Took me less than 30m to install it and get a post up :)
